frameworks/cassandra/tests/conftest.py

```diff
@@ -10,3 +17,21 @@ def configure_universe():
 @pytest.fixture(scope='session')
 def configure_security(configure_universe):
     yield from sdk_security.security_session('cassandra')
+
+@pytest.fixture(autouse=True)
+def get_cassandra_plans_on_failure(request):
+    yield
+    if sdk_utils.is_test_failure(request):
+        plans, err, rc = shakedown.run_dcos_command('cassandra plan list')
```

Comment:
This is not going to collect logs if we use foldered names for our services. Since this PR is already open, we could make the change now.

Reply:
(ok. Just realised that this is a backport. Please create a ticket to address this properly in both branches).

Reply:
Will do. Can you take off the request for change from this review?

Reply:
Done!